save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/inputform/InputWithDebounce.kt

```diff
@@ -89,35 +99,75 @@ external interface InputWithDebounceProps<T> : PropsWithChildren {
      * Maximum amount of options to display
      */
     var maxOptions: Int?
+
+    /**
+     * Flag that defines if input form is disabled or not
+     */
+    var isDisabled: Boolean?
 }
 
 /**
+ * Default renderer for [inputWithDebounceForUserInfo]
+ *
  * @param childrenBuilder [ChildrenBuilder] instance
  * @param userInfo user's [UserInfo]
  */
 internal fun renderUserWithAvatar(childrenBuilder: ChildrenBuilder, userInfo: UserInfo) {
     with(childrenBuilder) {
         div {
-            className = ClassName("row col d-flex align-items-center")
-            div {
-                className = ClassName("col-1")
-                img {
-                    className = ClassName("avatar avatar-user border color-bg-default rounded-circle pl-0")
-                    src = userInfo.avatar?.let { "/api/$v1/avatar$it" } ?: AVATAR_PROFILE_PLACEHOLDER
-                    style = jso {
-                        width = "2rem".unsafeCast<Width>()
-                        height = "2rem".unsafeCast<Height>()
-                    }
-                }
+            className = ClassName("row d-flex align-items-center")
+            renderAvatar(userInfo) {
+                width = 2.rem
+                height = 2.rem
             }
-            h4 {
+            h6 {
                 className = ClassName("col-auto mb-0")
                 +userInfo.name
             }
         }
     }
 }
 
+/**
+ * Default renderer for [inputWithDebounceForOrganizationDto]
+ *
+ * @param childrenBuilder [ChildrenBuilder] instance
+ * @param organizationDto [OrganizationDto] to display
+ */
+internal fun renderOrganizationWithAvatar(childrenBuilder: ChildrenBuilder, organizationDto: OrganizationDto) {
```

Comment:
I think this function is a duplicate. We already have a ```renderOrganizationWithName``` method

Reply:
This method has its signature and it is used for `inputWithDebounce`. it is nice to use it as `::renderOrganizationWithAvatar`.

Reply:
As you can see, this file contains three `render...` functions - all of them are used together with `inputWithDebounce`